homeassistant/components/fordpass/translations/en.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Don't edit the translations file directly. It should be generated using the script. See https://developers.home-assistant.io/docs/internationalization/core/